Failing CI tests related to analysis engine

```
  1) Activation - platform data
       Name and hash (Linux):
      AssertionError: linux-x64 does not match centos-x64
      + expected - actual
      -linux-x64
      +centos-x64
      
      at Object.<anonymous> (src/test/activation/platformData.test.ts:66:20)
      at Generator.next (<anonymous>)
      at fulfilled (out/test/activation/platformData.test.js:6:58)
      at <anonymous>
  2) Activation - platform data
       Module name:
      AssertionError: Microsoft.PythonTools.VsCode.VsCode does not match Microsoft.PythonTools.VsCode
      + expected - actual
      -Microsoft.PythonTools.VsCode.VsCode
      +Microsoft.PythonTools.VsCode
      
      at Object.<anonymous> (src/test/activation/platformData.test.ts:81:20)
      at Generator.next (<anonymous>)
      at /home/travis/build/Microsoft/vscode-python/out/test/activation/platformData.test.js:9:71
      at Promise (<anonymous>)
      at __awaiter (out/test/activation/platformData.test.js:5:12)
      at Context.test (src/test/activation/platformData.test.ts:72:36)
```

--- a/src/test/activation/platformData.test.ts
+++ b/src/test/activation/platformData.test.ts
@@ -15,18 +15,18 @@ const testDataWinMac = [
 ];
 
 const testDataLinux = [
-    { name: 'centos', expectedName: 'centos-x64' },
-    { name: 'debian', expectedName: 'debian-x64' },
-    { name: 'fedora', expectedName: 'fedora-x64' },
-    { name: 'ol', expectedName: 'ol-x64' },
-    { name: 'opensuse', expectedName: 'opensuse-x64' },
-    { name: 'rhel', expectedName: 'rhel-x64' },
-    { name: 'ubuntu', expectedName: 'ubuntu-x64' }
+    { name: 'centos', expectedName: 'linux-x64' },
+    { name: 'debian', expectedName: 'linux-x64' },
+    { name: 'fedora', expectedName: 'linux-x64' },
+    { name: 'ol', expectedName: 'linux-x64' },
+    { name: 'opensuse', expectedName: 'linux-x64' },
+    { name: 'rhel', expectedName: 'linux-x64' },
+    { name: 'ubuntu', expectedName: 'linux-x64' }
 ];
 
 const testDataModuleName = [
     { isWindows: true, expectedName: 'Microsoft.PythonTools.VsCode.exe' },
-    { isWindows: false, expectedName: 'Microsoft.PythonTools.VsCode' }
+    { isWindows: false, expectedName: 'Microsoft.PythonTools.VsCode.VsCode' }
 ];
 
 // tslint:disable-next-line:max-func-body-length

